.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+<!-- Do you have a question? Please ask it on https://stackoverflow.com/questions/tagged/monaco-editor -->
```

Comment:
# Issue Title

Be as descriptive as you can with your title.

# Description

**Expected behavior:** [What you expect to happen]
**Actual behavior:** [What actually happens]
**Reproduces how often:** [What percentage of the time does it reproduce?]

# General

Provide details on the problem you are experiencing including the .NET version, operating system version and anything else that is relevant.

# Code Sample

Please provide a code sample for your issue if it is relevant, either inline, link to a gist (or similar) or add it as a zipped attachment.

Reply:
I have added a good example of these docs for reference.  Feel free to use these as the basis of your PR.  I would suggest looking over them and making changes as you see fit.